identity/identity_test.go

```diff
@@ -154,7 +170,7 @@ func TestHandler_florenceToken(t *testing.T) {
 
 	Convey("Given a request with a florence token, and mock client that returns 200", t, func() {
 
-		req := httptest.NewRequest("GET", url, nil)
+		req := httptest.NewRequest("GET", url, bytes.NewBufferString("some body content"))
```

Comment:
forgot to check for draining within this test?

Reply:
I was debating if I should add a test in this scenario. The downstream handler is called, so the request body should not be drained in this case. I have added assert to check that it was not drained.